book/en/src/preface.md

```diff
@@ -141,7 +141,7 @@ Asynchronous programming in various forms are getting increased popularity and l
 
 The Rust standard library provides collections for dynamically allocated data-structures which are useful to manage execution contexts at run-time. However, in the setting of resource constrained real-time systems, dynamic allocations are problematic (both regarding performance and reliability - Rust runs into a *panic* on an out-of-memory condition). Thus, static allocation is the preferable approach!
 
-RTIC provides a mechanism for `async`/`await` that relies solely on static allocations. However, the implementation relies on the `#![feature(type_alias_impl_trait)]` (TAIT) which is undergoing stabilization (thus RTIC v2.x currently requires a *nightly* toolchain). Technically, using TAIT, the compiler determines the size of each execution context allowing static allocation.
+RTIC provides a mechanism for `async`/`await` that relies solely on allocations in `main` before `init` is called. The implementation does not rely on the `#![feature(type_alias_impl_trait)]` (TAIT) and thus RTIC v2.x **does not** require *nightly* toolchain. A stable Rust compiler from 1.75 is enough. 
```

Comment:
If someone didn't know that RTIC used TAIT before this remark sounds weird, why would someone care what RTIC does not need? It probably should mention since which specific RTIC version TAIT and nightly are not necessary